### PR TITLE
updated jupyterhub, spark, &  minio config

### PR DESCRIPTION
**Pull request by Nightbane112**
The following commits fixes : 

- Spark driver unable to resolve container hostname
- Jupyterhub worker containers crash loop. (Worker container crashes a few seconds after creation)

### CODE REVIEW

File: helm/elixier/templates/jupyterhub/statefulset.yaml [hunk 1]
```diff
@@ -58,6 +58,10 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.name
+            - name: K8S_NAMESPACE
```
Comment (kagesenshi) [inline helm/elixier/templates/jupyterhub/statefulset.yaml]: Can you remove this .. see if stuff still work?

File: helm/elixier/templates/spark3/_helpers/spark-defaults.conf.tpl [hunk 2]
```diff
@@ -39,7 +39,7 @@ spark.blockManager.port {{ .Values.spark.blockmanager_port | default "2044" }}
 spark.executor.cores {{ .Values.spark.executor_cores }}
 spark.executor.memory {{ .Values.spark.executor_memory }}
 
-
+spark.driver.host ${env:HOSTNAME}
```
Comment (kagesenshi) [inline helm/elixier/templates/spark3/_helpers/spark-defaults.conf.tpl]: Remove this and see if stuff still works or not

**Review by kagesenshi — COMMENTED** [review]
Please check remove some of the customization and see if still works or not .. I don't think they are necessary after we figured out the root cause.

File: helm/elixier/templates/jupyterhub/statefulset.yaml (hunk 1 above)
Comment (Nightbane112) [reply helm/elixier/templates/jupyterhub/statefulset.yaml]: Removed commit and its jupyterhub & spark works

File: helm/elixier/templates/spark3/_helpers/spark-defaults.conf.tpl (hunk 2 above)
Comment (Nightbane112) [reply helm/elixier/templates/spark3/_helpers/spark-defaults.conf.tpl]: Removed this commit too, in addition with this commit too [https://github.com/elixierdata/helm/pull/1/files/377e5c91dfa962cd8fec085e2544adf6feb9e461#r1068088943](https://github.com/elixierdata/helm/pull/1/files/377e5c91dfa962cd8fec085e2544adf6feb9e461#r1068088943)

Jupyter & spark are working

**Comment by kagesenshi** [comment]
Merged 

Thank you

**Pull request merged**